substreams/ethereum-uniswap-v4/src/modules/6_map_protocol_changes.rs

```diff
@@ -0,0 +1,228 @@
+use crate::pb::uniswap::v4::{
+    events::{pool_event, PoolEvent},
+    Events, LiquidityChanges, TickDeltas,
+};
+use itertools::Itertools;
+use std::{collections::HashMap, str::FromStr, vec};
+use substreams::{pb::substreams::StoreDeltas, scalar::BigInt};
+use substreams_ethereum::pb::eth::v2::{self as eth};
+use substreams_helper::hex::Hexable;
+use tycho_substreams::{balances::aggregate_balances_changes, prelude::*};
+
+type PoolAddress = Vec<u8>;
+
+#[substreams::handlers::map]
+pub fn map_protocol_changes(
```

Comment:
Did you change anything here compared to `uniswap_v3`?

Reply:
- Fee naming: protocol_fees/token0 -> protocol_fees/zero2one
- Match new event names
- Skip reparsing Initialize 
- Separate lower & upper bits from ProtocolFeeUpdated event

Reply:
Ok perfect, so no change in this map, only adapted `event_to_attributes_updates` to match USV4 things, right?